src/run.ts

```diff
@@ -12,11 +11,6 @@ import logger from './utils/logger'
 
 initialiseAppInsights(applicationInfo())
 
-const gatherComponentDependencies = async (env: Environment, components: ComponentInfo) => {
-  const dependencies: Dependency[] = await getDependencies(env)
-  return components.getKnownComponents().concat(dependencies)
-}
```

Comment:
this was a bit weird as needed to initialise all the known components first and treated them as fake dependencies - with an undefined target.

This is now all handled in side the `buildComponentMap` without having to create a load of pseudo dependencies.